ngraph/src/ngraph/builder/reshape.hpp

```diff
@@ -78,6 +78,7 @@ namespace ngraph
         ///
         /// \return     The squeezed node.
         ///
+        NGRAPH_API
```

Comment:
Why we expose this method?

Reply:
Because we use it in the code which was moved to tests.